WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -466,26 +451,32 @@ private async Task FailFastTransactionSigningPhaseAsync(Round round, Cancellatio
 
 		round.LogInfo($"Removed {removedAlices} alices, because they weren't ready. Remaining: {round.InputCount}");
 
-		if (round.InputCount >= round.Parameters.MinInputCountByRound)
-		{
-			EndRound(round, EndRoundState.NotAllAlicesSign);
-			await CreateBlameRoundAsync(round, cancellationToken).ConfigureAwait(false);
-		}
-		else
+		await EndRoundAndTryCreateBlameRoundAsync(round, cancellationToken).ConfigureAwait(false);
+	}
+
+	private async Task EndRoundAndTryCreateBlameRoundAsync(Round round, CancellationToken cancellationToken)
+	{
+		if (round.InputCount < Config.MinInputCountByBlameRound)
```

Comment:
This is wrong because you are comparing a round condition against a config value instead of doing it against the round parameter. This is the same that I mentioned here https://github.com/zkSNACKs/WalletWasabi/pull/12444#discussion_r1488140993

Perhaps this is the price to pay for a much less complex solution, and it can be okay if we don't abuse of this.